### PR TITLE
Feature/random address

### PR DESCRIPTION
**Pull request by cshcomcom**
## Description
- Now we can do random transfer test with `randomAddressPath` & `extraPath` configs!

## Screenshot
image

### CODE REVIEW

File: transfer_sample.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      "ainAddress": "0x00ADEc28B6a845a085e03591bE7550dd68673C1C",
+      "ainPrivateKey": "b22c95ffc4a5c096f7d7d0487ba963ce6ac945bdc91c79b64ce209de289bec96",
+      "transactionOperation": {
+        "type": "SET_VALUE",
+        "ref": "/transfer/0x00ADEc28B6a845a085e03591bE7550dd68673C1C",
+        "value": 1
+      },
+      "randomAddressPath": true,
+      "consecutivePath": true,
+      "extraPath": "value",
```
Comment (liayoo) [inline transfer_sample.json:18]: suggestion) how about pathSuffix?

**Review by platfowner — APPROVED** [review]
LGTM
Thanks!

**Review by liayoo — APPROVED** [review]
LGTM with a comment. Thanks!

File: transfer_sample.json (hunk 1 above)
Comment (cshcomcom) [reply transfer_sample.json:18]: Looks good! Done

**Comment by cshcomcom** [comment]
Thanks for reviews! Now merging

**Pull request merged**